docs/config-options.md

```diff
@@ -0,0 +1,65 @@
+# Config Options
+
+This file documents all the configuration options supported by the Spanner Cassandra Adapter.
+
+```yaml
+# [Optional] Global client configurations that apply to all listeners.
+globalClientConfigs:
+  # [Optional] The Spanner endpoint to use. Defaults to "spanner.googleapis.com:443".
+  spannerEndpoint: "spanner.googleapis.com:443"
```

Comment:
I would remove this option to avoid any confusion to end users, usually they copy-paste settings from the samples